frontend/components/PageComponentFactory/index.spec.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import pageComponentFactory from './index';
+
+describe('pageComponentFactory', () => {
+  // eslint-disable-next-line require-jsdoc, react/prop-types
+  const View = ({ children }) => (<div id="view">{children}</div>);
+  // eslint-disable-next-line require-jsdoc, react/prop-types, no-shadow
+  const ComponentProto = ({ View }) => (<View><div id="component">Hello world</div></View>);
+
+  it('should render a component in a view', () => {
+    const ComponentClass = pageComponentFactory({
+      View,
+      ComponentProto,
+    });
+    const component = mount(<ComponentClass />);
+
+    expect(component.find('#view').length).toBe(1);
```

Comment:
hint, you could do and save the .length
```suggestion
    expect(component.exists('#view')).toBe(true);
```

Reply:
True